demo/index.html

```diff
@@ -123,49 +128,70 @@ <h1>${this.title}</h1>
         `;
       }
     }
-
     customElements.define('menu-content', MenuContent);
 
-    const menuSlider = document.createElement('ia-menu-slider');
-    menuSlider.menus = [{
+    const menus = [{
       icon: html`
         <ia-icon icon="audio"></ia-icon>
       `,
+      title: 'Audio Menu',
+      menuDetails: '(100,000,000 tracks)',
+      actionButton: html`
+        <button
+          style='color: white;'
+          @click=${e => console.log('CLICK EXTRA ACTION', e)}
+        >
+          <ia-icon-volumes></ia-icon-volumes>
+        </button>
+      `,
       label: 'Audio',
       id: 'audio',
       component: html`
         <menu-content
-          .title=${"Audio menu"}
-          .items=${[1, 2, 3, 4].map((i) => 'Song ' + i)}
+          .title='Audio menu'
+          .items=${[1, 2, 3, 4].map(i => `Song ${i}`)}
         ></menu-content>
-      `
+      `,
     }, {
       icon: html`
         <ia-icon icon="software"></ia-icon>
       `,
+      title: 'Software Menu',
+      menuDetails: '(10 apps)',
       label: 'Software',
       id: 'software',
       component: html`
         <menu-content
-          .title=${"Software menu"}
-          .items=${[1, 2, 3, 4, 5, 6].map((i) => 'CD-ROM ' + i)}
+          .title='Software menu'
+          .items=${[1, 2, 3, 4, 5, 6].map(i => `CD-ROM ${i}`)}
         ></menu-content>
-      `
+      `,
     }, {
       icon: html`
         <ia-icon icon="donate" style="--iconFillColor: #f00"></ia-icon>
       `,
       label: 'Donate',
       id: 'donate',
       followable: true,
-      href: '#'
+      href: '#',
     }];
 
-    document.querySelector('section').appendChild(menuSlider);
-
-    document.querySelector('button').addEventListener('click', () => {
-      menuSlider.open = !menuSlider.open;
-    });
+    (function toggleScope() {
+      let menuOpen = false;
+      const closeCB = (event) => {
+        console.log('Drawer Closed', event);
```

Comment:
QUESTION: Did you want this debug info in here?

Reply:
hi yes, thanks for asking. it's for the demo.